### DISCUSSION TITLE
getting line 5: .endpoints.microsoftGraphResourceId: command not found error in set_ad_role.sh

### DISCUSSION THREAD

**Discussion by sunil8779kumar** [aztfmod/terraform-azurerm-caf#1264] [General]
hi @arnaudlh  , I am using all of yours code for building landing zones using terraform. I am able to build test env by using your code and that is super easy to understand, thanks for that !!!.#

Now i want to add some users to my test environment root management group via code using terraform module. i am referring your code frm https://github.com/aztfmod/terraform-azurerm-caf/tree/main/modules/azuread/roles. But when i run the terraform apply command, it says below error for set_ad_role.sh script

/set_ad_role.sh:
2022-07-19T18:14:47.9312391Z line 5: .endpoints.microsoftGraphResourceId: command not found
2022-07-19T18:14:47.9313061Z Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w'
2022-07-19T18:14:47.9313611Z encoding='utf-8'>
2022-07-19T18:14:47.9314143Z BrokenPipeError: [Errno 32] Broken pipe

any insights will be highly appreciated

**Comment by sunil8779kumar** [comment]
Hi @LaurentLesle , could you also please help me here? i am kind of stuck since two days, not able to find a way to fix this

**Reply by LaurentLesle** [reply]
Are you using the rover? It is used for some edge case scenario to call az cli.

**Reply by sunil8779kumar** [reply]
no I am not using rover, just trying to use it in my devops pipeline
